src/opencl/image_function_opencl.cpp

```diff
@@ -292,6 +293,25 @@ namespace
             }
         }
 
+        __kernel void setPixelOpenCL( __global const uchar * data, uint offset, uint rowSize, uint width, uint height, uint value )
```

Comment:
`__global const uchar * data` should be writable so `__global uchar * data`. The same applies to below function.

Reply:
We miss `x` and `y` as input parameters.

Reply:
Added the params, and removed const.